consensus/metrics.go

```diff
@@ -83,3 +90,15 @@ func (consensus *Consensus) UpdateLeaderMetrics(numCommits float64, blockNum flo
 	consensusCounterVec.With(prometheus.Labels{"consensus": "num_commits"}).Add(numCommits)
 	consensusGaugeVec.With(prometheus.Labels{"consensus": "num_commits"}).Set(numCommits)
 }
+
+func initMetrics() {
+	onceMetrics.Do(func() {
+		prom.PromRegistry().MustRegister(
```

Comment:
Is it possible that this function will probably panic? It is ok if we exit node program if this problem happens?

Reply:
it will panic when duplicated metrics are registered.  In our case, since we have static metrics created here and not dynamic metrics created, so the risk is very low. We should be able to detect this kind of panic in tests though.